scipy/stats/_stats_mstats_common.py

```diff
@@ -46,7 +46,8 @@ def linregress(x, y=None, alternative='two-sided'):
         intercept : float
             Intercept of the regression line.
         rvalue : float
-            Correlation coefficient.
+            The Pearson correlation coefficient. The square of ``rvalue``
+            is the coefficient of determination.
```

Comment:
This does indeed appear to be consistent with the suggestion from @charlotte12l in the matching issue--I'll let them confirm.

The CI failure looks unrelated.

Reply:
Thanks！ I think `is equal to` may be better?  "_The square of ``rvalue`` is equal to the coefficient of determination._"

Reply:
Yeah, I like that.